marconi-cardano-core/src/Marconi/Cardano/Core/Types.hs

```diff
@@ -118,6 +117,9 @@ type TxOutRef = C.TxIn
 txOutRef :: C.TxId -> C.TxIx -> C.TxIn
 txOutRef = C.TxIn
 
+data IndexingDepth = MinIndexingDepth !Word64 | MaxIndexingDepth
```

Comment:
Looks like this is added back in here after being removed in https://github.com/input-output-hk/marconi/pull/248. I assume it is just a git-goof, but let me know if not.

Reply:
Yep, this was me incorrectly resolving conflicts caused by my own work! I've re-removed it.